src/native/fs/fs_napi.cpp

```diff
@@ -1497,6 +1553,18 @@ FileWrap::linkfileat(const Napi::CallbackInfo& info)
     return api<LinkFileAt>(info);
 }
 
+Napi::Value
+FileWrap::unlinkfileat(const Napi::CallbackInfo& info)
+{
+    return api<UnlinkFileAt>(info);
+}
+
+Napi::Value
+FileWrap::getfd(const Napi::CallbackInfo& info)
+{
+    return api<GetFd>(info);
```

Comment:
you don't need to submit a worker to the threadpool to do this, FileWrap keeps the _fd and you can just return it synchronously here with `return Napi::Number::New(info.Env(), _fd);`

Reply:
Cool thanks, it's a temporary change, I wish to remove it when unlinkat will support AT_FDCWD flag